test/playwright/utils/navigation.ts

```diff
@@ -11,7 +11,57 @@ import {
 } from '~/constants/media'
 import { SCREEN_SIZES } from '~/constants/screens'
 
-import messages from '~/locales/en.json'
+import enMessages from '~/locales/en.json'
+import rtlMessages from '~/locales/ar.json'
+
+const messages: Record<string, Record<string, unknown>> = {
+  ltr: enMessages,
+  rtl: rtlMessages,
+}
+
+const getNestedProperty = (
+  obj: Record<string, unknown>,
+  path: string
+): string => {
+  const value = path
+    .split('.')
+    .reduce((acc: string | Record<string, unknown>, part) => {
+      if (typeof acc === 'string') {
+        return acc
+      }
+      if (Object.keys(acc as Record<string, unknown>).includes(part)) {
+        return (acc as Record<string, string | Record<string, unknown>>)[part]
+      }
+      return ''
+    }, obj)
+  return typeof value === 'string' ? value : JSON.stringify(value)
+}
+
+/**
+ * Simplified i18n t function that returns English messages for `ltr` and Arabic for `rtl`.
+ * It can also handle nested labels ('header.title').
+ * @param path - The label to translate.
+ * @param dir - The language direction.
+ */
+export const t = (path: string, dir: LanguageDirection = 'ltr'): string => {
+  let value = ''
+  if (dir === 'rtl') {
+    value = getNestedProperty(messages.rtl, path)
+  }
+  return value === '' ? getNestedProperty(messages.ltr, path) : value
```

Comment:
```suggestion
return getNestedProperty(messages[dir], path) ?? ''
```

what about something like this instead?

Reply:
We need to fall back to `ltr` (i.e., English) locale if we can't find the value in other locales. I could change `getNestedProperty` to return `null` if it doesn't find the key, and then replace the lines above with something like this:

```
let value = dir === 'rtl' ? getNestedProperty(messages.rtl, path) : null;
return value ?? getNestedProperty(messages.ltr, path) || '';
```
Would it be clearer?

Reply:
Oh I see, I was missing that context that we need to fallback to `ltr` if `rtl` isn't present. Of course! I think it's fine as-is, then.